WcaOnRails/app/views/competitions/show.html.erb

```diff
@@ -40,5 +40,6 @@
         <% end %>
       </div>
     <% end %>
+    <%= render "time_limit_div", competition: @competition %>
```

Comment:
This could definitely use better wording. Even replacing `_div` with `_info` would be an improvement. Perfectly it would somehow describe Time limit, Cutoff and Format, but I don't have any better idea than `time_limit_cutoff_format_info` ^^

Do whatever makes sense to you!